src/Avalonia.Controls/ComboBox.cs

```diff
@@ -173,6 +188,24 @@ public VerticalAlignment VerticalContentAlignment
             set { SetValue(VerticalContentAlignmentProperty, value); }
         }
 
+        /// <summary>
+        /// Gets or sets the data template used to display the item in the combo box (not the dropdown)
+        /// </summary>
+        public IDataTemplate? SelectedItemTemplate
```

Comment:
What does this property do? I can't see it used anywhere. What's the difference between this and `DisplaySelectedItemTemplate`?